src/shared/components/mutationMapper/MutationMapper.tsx

```diff
@@ -345,12 +345,19 @@ export default class MutationMapper<P extends IMutationMapperProps> extends Reac
 
     protected filterResetPanel(): JSX.Element|null
     {
+        const dataStore = this.props.store.dataStore;
+
         return (
-            <div style={{marginTop:"5px", marginBottom:"5px"}}>
-                <span style={{color:"red", fontSize:"14px", fontFamily:"verdana,arial,sans-serif"}}>
-                    <span>Current view shows filtered results. Click </span>
-                    <a style={{cursor:"pointer"}} onClick={this.handlers.resetDataStore}>here</a>
-                    <span> to reset all filters.</span>
+            <div className={classnames("alert" , "alert-success")}>
+                <span style={{verticalAlign:"middle"}}>
+                    {`${dataStore.tableData.length}/${dataStore.allData.length} mutations are shown based on your filtering.`}
```

Comment:
using `tableData` here to fix the count problem (previously it was `sortedFilteredData` which doesn't include the data selection from the lollipop)